vmm/src/lib.rs

```diff
@@ -3506,6 +3454,7 @@ mod tests {
     }
 
     #[test]
+    #[allow(clippy::cognitive_complexity)]
```

Comment:
It's a tricky metric this one - AFAIK, it basically tells you there's too much logic cramped inside a single function. Since I added one more line to the test function, I guess I just pushed the clippy detector over the edge.